<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial State</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="init_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="show1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="show9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Final State</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="final_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Begin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Display">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
